data/projects/debug_profile.md

```diff
@@ -1,26 +1,49 @@
-In the development lifecycle of a Bear it needs to be debugged and profiled
-and coala should provide a comprehensive environment
-for this. In this project the participant shall develop a system to easily
+A famous programming quote reads "Debugging is twice as hard as writing code
+in the first place". Thus even though writing code can be as trivial as
+opening up your favorite text editor and crunching some keys we need tools to
+help us debug our code. An essential toolset that coala lacks is the ability
+to debug it's bears i.e the analysis procedures. In the development lifecycle
+of a Bear it needs to be debugged and profiled and coala should provide a
+comprehensive environment for this.
+
+In this project the participant shall develop a system to easily
 step into the analysis code (likely using the Pdb module already integrated
-into Python) and step out as soon as the analysis finished.
+into Python) and step out as soon as the analysis finished. The following are desireable goals for such a toolset implementaion:
```

Comment:
The following lists defines features rather than implementation details, therefore "toolset implementaion" -> "toolset's features"